templates/placementapi/config/httpd.conf

```diff
@@ -28,22 +28,47 @@ TransferLog /dev/stdout
 CustomLog /dev/stdout combined env=!forwarded
 CustomLog /dev/stdout proxy env=forwarded
 
+{{ range $endpt, $vhost := .VHosts }}
+# {{ $endpt }} vhost {{ $vhost.ServerName }} configuration
```

Comment:
nit: is this some debugging leftover?

Reply:
this is intentional to have a comment in the httpd.conf for the virtual host. if it is considered to be useless we can remove it

Reply:
Ahh, at first I thought that this is a comment in the template, but actually it is a comment template that is expaneded by the config generation to get a comment in the resulting config file. So this is good. Keep it. :)